examples/systemcalls/systemcalls.c

```diff
@@ -92,8 +150,41 @@ bool do_exec_redirect(const char *outputfile, int count, ...)
  *   The rest of the behaviour is same as do_exec()
  *
 */
+    int fd = open(outputfile, O_WRONLY | O_CREAT | O_TRUNC, 0644);
```

Comment:
@AbhishekKoppaCU think about how you could avoid duplicating logic with the function above.